common-docs/python.md

```diff
@@ -0,0 +1,14 @@
+# Python
+
+The Microsoft MakeCode programming environment uses Python along with the [JavaScript](/javascript) language.
+
+These topics give a brief introduction to Python with MakeCode:
+
+* **[Calling](/python/call)** - How to use a function
+* **[Sequencing](/python/sequence)** - Ordering statements in code
+* **[Variables](/python/variables)** -  Make a way to remember data
+* **[Operators](/python/operators)** - Things to change and compare values
+* **[Statements](/python/statements)** - An elements of to code to take action
```

Comment:
Grammar seems off

Reply:
It was done intentionally for the JavaScript page quite some time ago as a more relaxed style thing.

https://makecode.microbit.org/javascript